libs/webb-ui-components/src/components/ConnectWalletMobileButton/ConnectWalletMobileButton.tsx

```diff
@@ -0,0 +1,64 @@
+import { forwardRef } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import { Typography } from '../../typography';
+import { Button, ButtonProps } from '../Button';
+import {
+  BottomDialog,
+  BottomDialogTrigger,
+  BottomDialogPortal,
+} from '../BottomDialog';
+import { ConnectWalletMobileButtonProps } from './types';
+
+const BRIDGE_LINK = 'webb.tools/hubble_bridge';
```

Comment:
Please consider moving this into the constants file in the `webb-ui-components` too @vutuanlinh2k2.